### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by akhildevelops**
Android SDK Tools and APT packages Installer

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -282,4 +282,9 @@
   maintainer: Swift Server Workgroup
   contact: https://github.com/swift-server/swift-devcontainer-template/issues
   repository: https://github.com/swift-server/swift-devcontainer-template
-  ociReference: ghcr.io/swift-server/swift-devcontainer-template
\ No newline at end of file
+  ociReference: ghcr.io/swift-server/swift-devcontainer-template
+- name: Android SDK Tools and Apt packages
+  maintainer: @akhildevelops
+  contact: https://github.com/akhildevelops/devcontainer-features/issues
+  repository: https://github.com/akhildevelops/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:289]: Looks like your published Features are private, the community won't be able to use them unless they are public. Mind switching the visibility? thanks! 

https://github.com/akhildevelops?tab=packages&repo_name=devcontainer-features

image

**Review by samruddhikhandale — COMMENTED** [review]
Thanks, can you help fix the merge conflict?

File: _data/collection-index.yml (hunk 1 above)
Comment (akhildevelops) [reply _data/collection-index.yml:289]: Made them public. Thanks for pointing out :handshake:

**Comment by eitsupi** [comment]
> APT packages

Hi @akhildevelops, I just wanted to make sure you knew that a Feature similar to this already exists.
https://github.com/rocker-org/devcontainer-features/tree/main/src/apt-packages

If it is a Feature with similar functionality but different purpose, it may be worth documenting.

**Comment by akhildevelops** [comment]
I haven't come across it. My feature also serves the same purpose.

**Review by samruddhikhandale — APPROVED** [review]
Nice, thanks!

**Pull request merged**